homeworks/dmytro.tolbatov_dmytrotolbatov/homework_5/sumClosure.js

```diff
@@ -0,0 +1,14 @@
+function sumClosure(arr) {
+    const result = [];
+    arr.reduce((sum, current) => {
+        result.push(sum + current);
+
+        return sum + current;
+    }, null);
```

Comment:
though coercion works fine here, I would provide `0` instead of `null` as a starting value for an accumulator